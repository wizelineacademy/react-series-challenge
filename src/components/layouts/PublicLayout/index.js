import React from "react";
import Header from "../../organism/Header/index";
import PropTypes from "prop-types";

const PublicLayout = ({ children }) => (
  <div style={{ height: "100%" }}>
    <Header />
    <div>{children}</div>
  </div>
);

PublicLayout.propTypes = {
  children: PropTypes.node
};

export default PublicLayout;
