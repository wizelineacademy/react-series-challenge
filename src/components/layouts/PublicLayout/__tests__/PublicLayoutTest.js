import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import PublicLayout from "../index";

describe("PublicLayout", () => {
  it("should render without crashing", () => {
    const testPublicLayout = shallow(<PublicLayout />);

    expect(shallowToJson(testPublicLayout)).toMatchSnapshot();
  });
});
