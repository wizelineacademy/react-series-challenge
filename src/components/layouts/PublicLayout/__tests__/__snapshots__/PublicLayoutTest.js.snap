// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PublicLayout should render without crashing 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <Header />
  <div />
</div>
`;
